_includes/sideblock.html

```diff
@@ -1,7 +1,9 @@
 <div class="sideblock-wrapper col-md-3 d-none d-lg-block">
   <div class="sideblock">
     {% if include.image %}
-    <img alt="" src="{{ include.image }}" style="max-width: 100%; margin-bottom: 10px">
+    <a href="{{ include.external_website }}" target="_blank">
+      <img alt="{{t.go_to_website}} {{ include.external_website }}" src="{{ include.image }}" style="max-width: 100%; margin-bottom: 10px">
```

Comment:
This will result in an empy url if `external_website` is not set for a specific platform. Probably it's better to add an extra check and include just the image in such a case, WDYT?